test/TensorFlowRuntime/dynamic_compilation_tensor_group.swift

```diff
@@ -107,11 +107,19 @@ TensorGroupTest.testAllBackends("output to empty struct, address-only") {
 // values.
 TensorGroupTest.testAllBackends("input, loadable") {
   let example = Example(x: Tensor([1, 2]), y: Tensor([3, 4]))
-  let packed: Tensor<Float> = #tfop("Pack", [example],
+  let packed: Tensor<Float> = #tfop("Pack", example,
                                     T$dtype: Float.tensorFlowDataType)
-  expectEqual(ShapedArray(shape: [2, 2], scalars: [1, 2, 3, 4]), packed.array)
+  expectEqual(ShapedArray(shape: [2, 2], scalars: [1, 2, 3, 4]),
+              packed.array)
 }
 
-// TODO(marcrasi): Loadable output tests, once that is implemented.
+TensorGroupTest.testAllBackends("output, loadable") {
+  let tensor = Tensor<Float>(shape: [2, 2], scalars: [1, 2, 3, 4])
+  let example: Example = #tfop("Unpack", tensor,
```

Comment:
I don't think that there's a way of doing that, because `UnsafePointer` is a pointer without any size information.

Reply:
Would using `UnsafeBufferPointer` here help?

Reply:
I think it's not worth the extra effort. We pass this function pointers that are generated in IRGen, so if we changed this function to accept UnsafeBufferPointers, then IRGen would have to construct the whole UnsafeBufferPointer struct instead of just a pointer.

Reply:
Agree that we should keep IRGen code simple. Indeed it's not clear such added safety is worth the complexity. I'm glad we assessed this and converged. Thanks!